CHANGELOG.md

```diff
@@ -7,22 +7,32 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 ### Added
+- Environment variable to run a custom initiialization script during CI testing: `CUSTOM_INIT_SCRIPT`
+- Environment variable to run from a subdirectory during CI testing: `USE_SUBDIR`
+- `assertComparativeEqual()` and `assertComparativeNotEqual()` to evaluate equality on an `a - b == 0` basis (and/or `!(a > b) && !(a < b)`)
+- `assertEqualFloat()` and `assertNotEqualFloat()` for comparing floats with epsilon
```

Comment:
When I'm feeling pedantic (which my family says is too often), I prefer "equivalent" to "equal".

Reply:
Sounds good, it removes the mismatch between "equal" and `<>`